src/js/parsers/section.js

```diff
@@ -69,14 +69,18 @@ class SectionParser {
 
     this._updateStateFromElement(element);
 
-    let childNodes = isTextNode(element) ? [element] : element.childNodes;
+    let finished = this.runPlugins(element);
```

Comment:
Given an `element` here that is a text node, wouldn't the add call here mean that text node will be passed to `this.runPlugins` twice? Once here and once in `pardeNode`.

Reply:
Good catch 😄 Pushed a fix